config/software/cmake.rb

```diff
@@ -24,7 +25,7 @@
 
 if windows?
   if windows_arch_i386?
-    # version_list: url=https://cmake.org/files/v3.20/ filter=*-windows-i386.zip
+    # version_list: url=  v3.20/ filter=*-windows-i386.zip
```

Comment:
Is there a reason to remove the URL here?

Reply:
I have fixed that typo error